src/server/scripts/EasternKingdoms/zone_undercity.cpp

```diff
@@ -157,15 +159,28 @@ class npc_lady_sylvanas_windrunner : public CreatureScript
                 if (LamentEventTimer <= diff)
                 {
                     DoSummon(ENTRY_HIGHBORNE_BUNNY, me, 10.0f, 3000, TEMPSUMMON_TIMED_DESPAWN);
-
+                    if (!flag) {
+                        Creature * sunsorrow = me->FindNearestCreature(AMBASSADOR_SUNSORROW, 40);
+                        if (sunsorrow && _player) {
+			    std::stringstream ss;
+			    ss << "That could have gone better ";
+			    ss << _player->GetName();
+			    ss << ", come here, please. I have something I need you to take care of for me.";
+			    sunsorrow->Whisper(ss.str(), LANG_UNIVERSAL,  _player);
```

Comment:
This should use `broadcast_text` id 12157 and not a blatantly hack constructing the string by hand.